system/lib/libc/musl/arch/emscripten/atomic_arch.h

```diff
@@ -99,88 +98,6 @@ static inline void a_store(volatile int *p, int x)
 {
 	__c11_atomic_store((_Atomic int*)p, x, __ATOMIC_SEQ_CST);
 }
-#else // __EMSCRIPTEN_PTHREADS__
-#define a_store_l a_store_l
```

Comment:
What are these `#define`s for? Could we get rid of those as well?

Reply:
These seem to appear in all the atomic_arch.h files in musl (one per arch).   I'm not sure of the reason but it makes sense to follow the norm I guess.